From ccb344967e5fa0616cdd8fded929612b01121997 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Wed, 27 Dec 2023 19:51:45 +0000
Subject: [PATCH 0804/1085] fixup downstream patch post driver conversion to
 CCI_REG

HTS was still using the raw register ID.

Fixes: dd26d43ddb7f ("media: i2c: imx219: make HBLANK r/w to allow longer exposures")
Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 drivers/media/i2c/imx219.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/media/i2c/imx219.c
+++ b/drivers/media/i2c/imx219.c
@@ -88,7 +88,7 @@
 /* HBLANK control range */
 #define IMX219_PPL_MIN			3448
 #define IMX219_PPL_MAX			0x7ff0
-#define IMX219_REG_HTS			0x0162
+#define IMX219_REG_HTS			CCI_REG16(0x0162)
 
 #define IMX219_REG_LINE_LENGTH_A	CCI_REG16(0x0162)
 #define IMX219_REG_X_ADD_STA_A		CCI_REG16(0x0164)
